src/pytest_plugins/consume/hive_simulators/conftest.py

```diff
@@ -42,6 +84,15 @@ def fixture_description(
         description += "\n\nNo description field provided in the fixture's 'info' section."
     else:
         description += f"\n\n{blockchain_fixture.info['description']}"
+    description += (
+        f"\n\nCommand to reproduce entirely in hive:" f"\n<code>{hive_consume_command}</code>"
+    )
+    description += (
+        f"\n\nCommands to reproduce within EEST using a hive dev back-end:"
+        f"\n1. <code>{eest_consume_commands[0]}</code>"
+        f"\n2. <code>{eest_consume_commands[1]}</code>"
```

Comment:
Use `sum` here instead of hard-coding indexes.

Reply:
Sounds good!